src/app/reducers/index.ts

```diff
@@ -85,9 +87,14 @@ export const getSpeechState = (state: State) => state.speech;
 
 export const getSearchResults = createSelector(getSearchState, fromSearch.getsearchresults);
 export const getItems = createSelector(getSearchState, fromSearch.getItems);
+export const getResponseTime = createSelector(getSearchState, fromSearch.getresponsetime);
 export const getTotalResults = createSelector(getSearchState, fromSearch.getTotalResults);
 export const getNavigation = createSelector(getSearchState, fromSearch.getNavigation);
+export const getSearchStatus = createSelector(getSearchState, fromSearch.getSearchstatus);
+
```

Comment:
Why there is an extra line?

Reply:
@samagragupta Probably because of tslint.